pkg/apis/servicecatalog/v1alpha1/zz_generated.defaults.go

```diff
@@ -28,5 +28,31 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
```

Comment:
I don't know how the code gen works.

Is this keying off the existence of the `defaults.go` file? There is no codegen annotation in it.